src/azure-cli/setup.py

```diff
@@ -179,6 +179,10 @@
             'template/service/template.json',
             'template/service/parameter.json'
         ],
+        'azure.cli.command_modules.appservice': [
+            'resources/WindowsFunctionsStacks.json'
+            'resources/LinuxFunctionsStacks.json'
```

Comment:
```suggestion
            'resources/WindowsFunctionsStacks.json',
            'resources/LinuxFunctionsStacks.json'
```
Missing a comma and they are considered to be concated into one file path `resources/WindowsFunctionsStacks.jsonresources/LinuxFunctionsStacks.json`.